src/main/java/seedu/duke/manager/StaffManager.java

```diff
@@ -103,4 +120,16 @@ public void deleteByStaffId(int staffId) {
     public int getNumOfStaffs() {
         return staffs.size();
     }
-}
+
+    @Override
+    protected void loadData() throws Exception {
+        this.staffs = new ArrayList<>();
+        ArrayList<?> list = (ArrayList<?>) this.load();
+        this.setStaffs((ArrayList<Staff>) list);
```

Comment:
you will need to loop through the array list, then cast to the actual object (in this case, `Staff`) and add individually. This helps to make sure that an exception will be raised here and not anytime in the future if you want to access the list and suddenly the object is not of the correct type.

Reply:
Okay